src/os/filestore/FileJournal.cc

```diff
@@ -1038,13 +1041,13 @@ void FileJournal::align_bl(off64_t pos, bufferlist& bl)
 {
   // make sure list segments are page aligned
   if (directio && (!bl.is_aligned(block_size) ||
-		   !bl.is_n_align_sized(CEPH_MINIMUM_BLOCK_SIZE))) {
-    assert(0 == "bl should be align");
-    if ((bl.length() & (CEPH_MINIMUM_BLOCK_SIZE - 1)) != 0 ||
-	(pos & (CEPH_MINIMUM_BLOCK_SIZE - 1)) != 0)
+		   !bl.is_n_align_sized(CEPH_DIRECTIO_ALIGNMENT))) {
+    if ((bl.length() & (CEPH_DIRECTIO_ALIGNMENT - 1)) != 0 ||
+	(pos & (CEPH_DIRECTIO_ALIGNMENT - 1)) != 0)
       dout(0) << "rebuild_page_aligned failed, " << bl << dendl;
-    assert((bl.length() & (CEPH_MINIMUM_BLOCK_SIZE - 1)) == 0);
-    assert((pos & (CEPH_MINIMUM_BLOCK_SIZE - 1)) == 0);
+    assert((bl.length() & (CEPH_DIRECTIO_ALIGNMENT - 1)) == 0);
+    assert((pos & (CEPH_DIRECTIO_ALIGNMENT - 1)) == 0);
+    assert(0 == "bl should be align");
```

Comment:
"aligned", while we're at it, and I think it's clearer to say "bl is not aligned"